adafruit_platformdetect/constants/boards.py

```diff
@@ -324,7 +324,7 @@
 # BeagleBone eeprom board ids from:
 #   https://github.com/beagleboard/image-builder
 # Thanks to zmatt on freenode #beagle for pointers.
-_BEAGLEBONE_BOARD_IDS = {
+_BEAGLEBONE_BOSARD_ID = {
```

Comment:
Hi, it looks like you accidentally changed the name here?

Reply:
Yes I think its Fixed

Reply:
it is not fixed

Reply:
Thanks for letting me learn how to correct my corrections.  I think we are good now